.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 repos:
 - repo: https://github.com/Yelp/detect-secrets
-  rev: v1.2.0
+  rev: v1.4.0
```

Comment:
What does this version bump mean/do?

Reply:
Every commit we do is scanned via detect secrets to help ensure we don't accidentally commit any secrets.

Full details of how to set it up are in the dev guide -> https://github.com/DEFRA/ffc-development-guide/blob/main/guides/developer-laptop-setup/install-detect-secrets.md

Detect secrets has had been updated recently and we needed to update this repo to that later version so effectively mark false positives.

Reply:
Thanks!